lib/services/commands/commandRegistryMongoDB.js

```diff
@@ -185,20 +185,16 @@ function removeFromDate(creationDate, callback) {
     if (Command.model) {
         Command.model.find(query).exec(function(error, commandList) {
             if (!error && commandList && commandList.length > 0) {
-                Command.model.remove(query, function(error, commandResult) {
+                Command.model.remove(query, function(error) {
```

Comment:
I guess the change here is motivated due to the change in some dependency library... Could you elaborate a little bit, pls?

Reply:
You're right, this was a change needed due to the update in Mongoose.

As you can see in https://mongoosejs.com/docs/api.html#model_Model.remove, the callback only returns the error parameter but no the result anymore.

Reply:
NTC